script/gen_requirements_all.py

```diff
@@ -232,7 +232,6 @@ def gather_modules():
     if errors:
         print("******* ERROR")
         print("Errors while importing: ", ', '.join(errors))
-        print("Make sure you import 3rd party libraries inside methods.")
```

Comment:
script will fail if you import from top for now.

Reply:
For integrations we use the integration loader. It only tries to import Python modules inside `homeassistant.script` and `homeassistant.auth`. They still rely on `REQUIREMENTS`

Reply:
@balloob see: #23488